project_task_pull_request/tests/test_project_task_pull_request.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Specialty Medical Drugstore
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestProjectTaskPullRequest(TransactionCase):
+    post_install = True
+    at_install = False
+
+    def setUp(self):
+        super(TestProjectTaskPullRequest, self).setUp()
+
+        project_obj = self.env["project.project"]
+        task_obj = self.env["project.task"]
+
+        self.project_1 = project_obj.create(
+            {"name": "Test Project 1", "pr_required_states": [(4, 6, 0)]}
```

Comment:
Are this and other numeric `id` references correct for the tests, instead of using `ref`?

Reply:
@Saran440 this question is relevant

Reply:
@dalonsod @pedrobaeza Fixed. Thank you for reviewed and remind me